composer.json

```
@@ -2,25 +2,25 @@
   "name": "nuvoleweb/robo-config",
   "description": "CI/developer-friendly configuration processor for Robo.",
   "type": "robo-tasks",
-  "license": "GPL-2.0",
+  "license": "GPL-2.0-or-later",
```

```suggestion
  "license": "GPL-2.0+",
```
as in another nuvole projects.

---

$ `composer validate`
> ./composer.json is valid, but with a few warnings
See https://getcomposer.org/doc/04-schema.md for details on the schema
# General warnings
- License "GPL-2.0+" is a deprecated SPDX license identifier, use "GPL-2.0-or-later" instead